### ISSUE TITLE
CM - Technical Specifications

### ISSUE THREAD

**Issue by eprocurementontology** [OP-TED/ePO#195]
To verify if the accessibility criteria should be here or whether it should be in Procurement, Award, Selection or Participation Criterion.

Keeping the accessibility criteria herein we put an  extra burden on the developers as they probably would expect them linked to the "Strategic Procurement" class instead, which is the place where the eForms puts it.

**Comment by muricna** [comment]
#191
Note the accessibility criteria mentioned in the eForms relates to whether the requested services, works, supplies takes into consideration accessibility requirements this is distinct from award critera.
